tools/flash_shelly.py

```diff
@@ -0,0 +1,499 @@
+import getopt, importlib, platform, urllib, json, logging, re
+from sys import argv, exit
```

Comment:
don't use from foo import blah, import foo and use foo.blah

Reply:
ok, guess memory is not an issue.

Reply:
```
from importlib import util
from sys import argv
```

these 2 are required, they can not be full imports. else modules is not found.

```
Andrews-MacBook-Pro:tools andrewblackburn$ python3 flash_shelly.py -la
Traceback (most recent call last):
  File "flash_shelly.py", line 68, in <module>
    if not importlib.util.find_spec("zeroconf"):
AttributeError: module 'importlib' has no attribute 'util'

Andrews-MacBook-Pro:tools andrewblackburn$ python3 flash_shelly.py -la
  File "flash_shelly.py", line 400
    def app(sys.argv):
               ^
SyntaxError: invalid syntax
```

Reply:
https://github.com/mongoose-os-apps/shelly-homekit/pull/176/commits/a39399332926dd2f642773d4765e5c36df3396fa
https://github.com/mongoose-os-apps/shelly-homekit/pull/176/commits/5b81af09f69a18d8fde32f0891a5b4381ec425d2

Reply:
https://github.com/mongoose-os-apps/shelly-homekit/pull/176/commits/f9c90340cd36225fb2190d643fb8628d2b882543